prow/config.yaml

```diff
@@ -1638,6 +1638,8 @@ presubmits:
         - "--"
         - "--build=dind"
         - "--deployment=dind"
+        - "--ginkgo-focus=[Conformance]"
+        - "--ginkgo-skip=(DNS)|(NodeSelector)"
```

Comment:
this seems fine for the presubmit, can you update the `ci-kubernetes-dind-conformance` job to include all tests (no skip)?

Reply:
er all conformance

Reply:
Hmm, well in any case, let's configure the "-conformance" job not to skip any but definitely skip known-broken ones in the presubmit :+1: